docs/user_guide/installation_and_version_guide/v0.13.0.rst

```diff
@@ -12,3 +12,8 @@ Other improvements
 * Vectorized the curve interpolation step in :code:`ThresholdOptimizer` to improve performance and readability: :pr:`1448` by :user:`Tahar Allouche <taharallouche>`.
 * Allowed multi-dimensional input data :code:`X` (:code:`ndims > 2`) in :code:`ThresholdOptimizer` and :code:`ExponentiatedGradient`: :pr:`1470` by :user:`Tahar Allouche <taharallouche>`.
 * Removed support for python 3.8 and added support for python 3.12: :pr:`1488` by :user:`Tamara Atanasoska <tamaraatanasoska>`.
+
+
+Notes
+----
+* Fairlearn can be now used with the latest versions of :code:`scikit-learn` and :code:`xgboost`: :pr:`1517` by :user:`Tamara Atanasoska <tamaraatanasoska>`.
```

Comment:
I don't mind this here, but this isn't really a change on the fairlearn side.

Reply:
I wasn't sure where folks would look for information like this 🤔 or people first just try, which would probably be what I'd do anyway :) we an remove it.